src/core/Akka.Streams/Implementation/Fusing/Ops.cs

```diff
@@ -2663,12 +2664,12 @@ private void PushOne()
                                     break;
                         
                                 default:
-                                    throw new AggregateException($"Unknown SupervisionStrategy directive: {strategy}", result.Exception);
+                                    throw new ArgumentOutOfRangeException($"Unknown SupervisionStrategy directive: {strategy}", result.Exception);
                             }
                             continue;
                         }
```

Comment:
Since `SetElement()` always get invoked in the same thread now, does that mean that this bit of code isn't needed anymore?

Reply:
The whole error handling code inside `PushOne()` I mean

Reply:
I think we decided in Slack that we probably still need error-handling in both places since there can be a delay on `FailStage` taking effect in the `AsyncCallback` - we'd still want to check for it whenever else we're calling `PushOne`